java/vector/src/test/java/org/apache/arrow/vector/ipc/MessageSerializerTest.java

```diff
@@ -154,9 +152,6 @@ public void testSchemaDictionaryMessageSerialization() throws IOException {
     assertEquals(schema, deserialized);
   }
 
-  @Rule
-  public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
Is this code really unused?

Reply:
The test that used it was added 7 years ago and since removed, so it appears so.